
# Articles to read 


Gruber TR. Toward principles for the design of ontologies used for knowledge sharing? Int J Hum Comput Stud. 1995;43:907–28

This is a collection of resources to be read and parsed.

https://janzz.technology/ontology-and-taxonomy-stop-comparing-things-that-are-incomparable/



Neves M, Damaschun A, Mah N, Lekschas F, Seltmann S, Stachelscheid H, Fontaine JF, Kurtz A, Leser U. Preliminary evaluation of the CellFinder literature curation pipeline for gene expression in kidney cells and anatomical parts. Database 2013, 2013:bat020. doi:10.1093/database/bat020

https://bmcbioinformatics.biomedcentral.com/articles/supplements/volume-18-supplement-17


flowCL: ontology-based cell population labelling in flow cytometry

CLO: The cell line ontology


Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain

In 1957, C.H. Waddington proposed a powerful metaphor for how differentiated cell types emerge from a single, totipotent cell (Waddington 1957). T

Searching large-scale scRNA-seq databases via unbiased cell embedding with Cell BLAST


Unifying single-cell annotations based on the Cell Ontology

Cell type nomenclature for the mammalian brain: Development and application of a systematic, extensible convention

https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3844401/

https://github.com/kevinrue/Hancock

DE QUEIROZ, K.  1994.  Replacement of an essentialistic perspective on taxonomicdefinitions as exemplified by the definition of "Mammalia."  Syst. Biol. 43: 497-510
 
  Replacement of an Essentialistic Perspective on Taxonomic Definitions as Exemplified by the Definition of "Mammalia"
https://www.sciencedirect.com/science/article/abs/pii/S1369848601000036?via%3Dihub

https://www.pnas.org/content/105/51/20067

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2662812/

## Molecular taxonomy of major neuronal classes in the adult mouse forebrain 
\cite{sugino2006molecular

/Genes and homology in nervous system evolution: Comparing gene functions, expression patterns, and cell type molecular fingerprints   \cite{arendt2005genes

Musser, J. M. & Wagner, G. P. Character trees from transcriptome data: origin and individuation of morphological characters and the so-called ‘species signal’. J. Exp. Zool. 324, 588–604 (2015).
Introduces the phenomenon of concerted transcriptome evolution and shows how it influences the comparison of cell type transcriptomes

Mullen, A. C. et al. Master transcription factors determine cell-type-specific responses to TGF-ß signaling. Cell 147, 565–576 (2011). Demonstrates empirically that cell type terminal selectors are responsible for mediating the cell type-specific response to a common cellular signal.

Hobert, O. Terminal selectors of neuronal identity. Curr. Top. Dev. Biol. 116, 455–475 (2016). Outlines the important role of terminal selector genes in establishing and maintaining cell type identity.


Stefanakis, N., Carrera, I. & Hobert, O. Regulatory logic of pan-neuronal gene expression in C. elegans. Neuron 87, 733–750 (2015). A systematic investigation of how different neuronal cell types use both cell type-specific and more general regulatory information to express pan-neuronal genes.

Arendt, D. Genes and homology in nervous system evolution: comparing gene functions, expression patterns, and cell type molecular fingerprints. Theory Biosci. 124, 185-197 (2005). A conceptual review that discusses the different levels of homology in evolutionary developmental biology research; it introduces the concept of molecular fingerprints for cell type comparisons.
Up next: 

% https://www.the-scientist.com/cover-story/advancing-techniques-reveal-the-brains-impressive-diversity-30197
% https://www.sciencedirect.com/topics/medicine-and-dentistry/cell-type
% http://www.diva-portal.org/smash/get/diva2:942241/FULLTEXT01.pdf
% https://sciencing.com/specialized-cells-definition-types-examples-13718073.html
% https://www.nature.com/scitable/blog/bio2.0/discovering_new_cell_types_one/

https://senselab.med.yale.edu/NeuronDB

https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000691

# Discovery of new cell types:

JJordao MJC, Sankowski R, Brendecke SM, Sagar LG, Tai YH, Tay TL, Schramm E, Armbruster S, Hagemeyer N, et al. Single-cell profiling identifies myeloid cell subsets with distinct fates during neuroinflammation. Science 2019;363(6425):eaat7554. https://doi.org/10.1126/science.aat7554.
23. Gan Y, Li N, Zou G, Xin Y, Guan J. Identification of cancer subtypes from single-cell RNA-seq data using a consensus clustering method. BMC Med Genet. 2018;11(Suppl 6):117.
24. Mrdjen D, Pavlovic A, Hartmann FJ, Schreiner B, Utz SG, Leung BP, Lelios I, Heppner FL, Kipnis J, Merkler D, et al. High-dimensional single-cell mapping of central nervous system immune cells reveals distinct myeloid subsets in health, aging, and disease. Immunity. 2018;48(2):380–95 e386.
25. Der E, Ranabothu S, Suryawanshi H, Akat KM, Clancy R, Morozov P, Kustagi M, Czuppa M, Izmirly P, Belmont HM, et al. Single cell RNA sequencing to dissect the molecular heterogeneity in lupus nephritis. JCI Insight. 2017;2(9).

26. Villani AC, Satija R, Reynolds G, Sarkizova S, Shekhar K, Fletcher J, Griesbeck M, Butler A, Zheng S, Lazo S, et al. Single-cell RNA-seq reveals new types of human blood dendritic cells, monocytes, and progenitors. Science. 2017; 356(6335


Stevens, C.F. (1998). Neuronal diversity: Too many cell types for comfort