## Heterogeneity of neutrophils
    [@doi:10.1038/s41577-019-0141-8]
    
### Highlights

Immunity functions through the concerted actions of diverse cell types with specialized tasks

% Cell types or cells??
Hence, a central goal of modern immunology has been to catalogue cells in an effort to unify observations, generate hypotheses and propose basic biological principles

However, ‘naming’ subsets creates rules and restrictions that typically prevent capturing the true biology of a cell. This observation is particularly evident for‘plastic’ immune cells, for which the capacity to adapt to environmental changes is a defining property


Neutrophils are traditionally defined as a type of myeloid cell with a short half- life, specific nuclear morphology, defined granule content and surface expression of markers, such as the glycosylphosphatidylinositol (GPI)-linked
receptor Ly6G in mice or CD66b in humans.

% Conceptual definition which cannot be applied in practice: how to know that any given cell matches these characteristics??

neutrophils have been described in a variety of 'flavours'
% Simply different cell types

In this Opinion article, we present
a critical discussion of neutrophil heterogeneity

Neutrophils are the most abundant circulating leukocytes in humans.

Historically, granulocyte precursors in humans have been defined from density gradients followed by histological inspection of the different fractions upon Giemsa staining9

Three neutrophil subsets within the mouse bone marrow: a committed proliferative precursor, termed a pre- neutrophil, which sequentially differentiates into non- proliferating immature and mature neutrophils.

In addition to the various types of immature neutrophils at different stages of maturation, the bone marrow is also a site for recycling of circulating neutrophils, at least in mice. Indeed, a large fraction of mature neutrophils that have aged in the circulation are recruited back to the bone marrow with circadian frequency.

Under steady-state conditions, neutrophils and other leukocyte subsets are released from the bone marrow and circulate for about half a day before infiltrating tissues.

Although the relatively short time of neutrophils in the blood would suggest homogeneous properties of these cells, marked diurnal changes in phenotype do occur, a phenomenon referred to as neutrophil ageing.

Notably, these features are significantly blunted in the early morning in mice, when neutrophils are recently released from the bone marrow. This finding suggests that neutrophils adjust their functions to the changing demands of the day — for example, to protect from microbial invasions during the animal’s active phase (when the exposure to pathogens is highest) or to exert reparative functions during the resting phase. 

% That is of utmost importance! How many articles state in which circadian moment an statement is valid?
% Whenever the times of collection are not randomized, this introduces systematic bias! 

at circadian clock genes also regulate the diurnal variations in phenotype and function in a cell- intrinsic manner34
through a process similar to that reported for inflammatory monocytes40

Do tissue- specific neutrophils exist?

% Obviously 

marginal zoneneutrophils in the human spleen adopt unique B cell-stimulating properties through the secretion of cytokines, chemoattractants and the pattern recognition receptor pentraxin 3 that stimulate class switch recombination and immunoglobulin production by B cells that reside in this region54,55
.

### Comments

References fluctuate between human, mouse or general "neutrophils".

## Neuronal cell types

    [@doi:10.1016/j.cub.2004.06.035]

### Highlights

Identifying the functionally distinct types of neuron is central to any bottom-up understanding of how the brain works.

How are neuronal types distinguished and why do neurobiologists care so much about them?

What do we mean by a cell type? This question has generated much tedious discussion, but the ultimate goal is simple — to find a way to single out a group of neurons that carry out a distinct task.

Increasingly, cell types can be distinguished by their expression of genes and/or proteins. Occasionally they are first distinguished by characteristic patterns of electrical activity. But the commonest way is the shape of the cell

For convenience, neuroanatomists from Cajal’s time to the present have given them evocative nicknames, as shown in Figure 1B, but this is only a mnemonic device.

The names have varying degrees of exactness and currency, ranging from the famously distinctive Purkinje cell to many lesser, poorly defined cells. Like genes, some cells appear under several names.

A particularly unfortunate piece of vagueness pertains to the hierarchy of groups. Terms like ‘variety’, ‘class’, ‘type’, ‘subclass’, and ‘subtype’ are used indiscriminately. The technically correct usage is perhaps for a ‘class’ to represent a collection of ‘types’ that share a common feature. In this usage there is no place for a ‘subclass’—indeed, there is probably a need for more taxonomic levels. The term ‘type’ is sometimes reserved for the terminally differentiated level, and that usage will be followed here.

The term ‘pyramidal cell’ is used for neurons in several brain structures (there are
hippocampal pyramidal cells) and generally denotes a large cell with a roughly triangular soma from which arise distinct sets of apical and basal dendrites.

t is believed that many cell types exist within the large class of neocortical pyramidal cells, but there is at present no powerful and universally agreed way to distinguish them

A mammalian retina contains five major classes of neurons divided into ~60 individual types.

The intrinsic neurons are the horizontal cells (2 types), bipolar cells (~12 types) and amacrine cells (~30 types).

Finally, individual types of retinal neuron exhibit regular spacing, such that neurons of a defined type maintain a minimum distance from other neurons of the same type.

For the first time in any substantially complex CNS structure, the inventory of cell types in the retina appears to be virtually complete.


### Comments

## Cell type nomenclature for the mammalian brain: Development and application of a systematic, extensible convention
    [@url:https://arxiv.org/abs/2006.05406]
    
### Highlights
To facilitate cross-dataset alignment and comparison, we created a working nomenclature convention for matching and tracking cell types across studies.
It augments but does not change existing cell type names provided in original publications and can be directly applied to data from new or published studies.
Finally, we propose conventions for assigning accurate yet flexible cell types names in the mammalian cortex as a step towards a much larger community-wide effort to organize multi-source, data-driven information related to cell types.

These distinct morphological and electrophysiological features have provided the basis for robust and consistent classifications of neuronal cell types.

The use of scRNA-seq, in particular, for cell type classification has increased exponentially since its introduction a decade ago (Tang et al., 2009), with approximately 2,000 published studies using such technology, and several hundred tools for data analysis (Zappia et al., 2018)

Studies combining scRNA-seq with traditional morphological and electrophysiological characterizations have found a more complicated relationship in the brain than in retina, with cell types defined by morphology and electrophysiology sometimes containing cells from several cell types defined using gene expression (Gouwens et al., 2020), and some transcriptomically-defined types containing cells with multiple morphologies (Hodge et al., 2020, 2019).

Within a single study, cell types need to be quantitatively defined, organized, named, and annotated by comparison to published work and cell metadata.

- how do we quantitatively define cell types in an individual study

For essentially all published studies, clusters (or provisional cell types; see below) are defined.

Resulting clusters are often presented as “cell types,” though likely represent a combination of cell types, cell states (such as cell cycle or cell activation) (Kotliar et al., 2019), and the intersection of discrete cell types with more continuous variation.

In some cases, clusters can then be linked with well-established historical types captured in a cell type ontology, or a structured controlled vocabulary for grouping information,

- how do we annotate cell types using experimental and historical context?


- how do we compare and match cell types identified in different studies?

- how should those cell types be named in a way that provides historical or comparative context?

some investigators use generic names like “interneuron 1” or “Ex1” and then use figures, tables, or text to link these clusters to historical cell types.
Others advocate for chaining critical cell type features in the name itself, resulting in names like “Neocortex M1 L6 CT pyramidal, Zfpm2 non-adapt GLU”
Allen Institute assigns semi-automated names that reference marker genes and metadata from a given dataset, such as “Exc L5–L6 THEMIS C1QL3” in human and “L5 IT ALM Pdf5” in mouse cortex, respectively

- what data/metadata infrastructure is required to achieve these goals of cell type classification in a scalable way? 

Any solution devised to tackle this question should ideally be user-friendly, so people will actually use it, and should directly address some of the ongoing challenges of ontology, data matching, and cell type naming described above in its implementation, providing some amount of immediate standardization of any cell type classifications included therein.

To address the latter two challenges and as a practical need to organize vast amounts of multimodal data from the Allen Institute and collaborators, we have developed a nomenclature convention.

At the core of the proposal are two key concepts: a taxonomy (defined above and in Table 1), which must be generated prior to implementation of this schema, and a cell set, which can represent any collection of cells within a specific taxonomy. Provisional cell types (sometimes called cell types for convenience) are particularly important cell sets that represent quantitatively- derived data clusters defined by the classification algorithm that generated the taxonomy.

Analysis of each dataset will create a taxonomy of cell clusters that needs to be tracked.

Each taxonomy is assigned a unique taxonomy id of the format CS[YYMMDD][#], where “CS” stands for cell set; Y, M, and D represent year, month, and day; and # is an index for compiling multiple taxonomies on a single day.

Taxonomies can be defined using a single dataset or through alignment of multiple datasets, and high confidence, high resolution taxonomies can be compiled into a reference taxonomy.

Cell set accession IDs track unique cell sets across the entire universe of taxonomies and are defined as [taxonomy id][unique # within taxonomy].
Cell set labels provide general tags that allow easy tracking of groups of more broadly-defined cell types (e.g., “neuron”) within large taxonomies, and are defined as [Canonical cell type][unique # within canonical cell type].

Cell set aliases include any number of cell set descriptors intended for public consumption, including any cell type names used in manuscripts.

Each cell set can have at most one preferred alias, which is a biologically-driven term that is selected from a controlled vocabulary, and that only can be assigned for cell sets with confident matches across taxonomies

- Provisional cell type: Quantitatively-derived data cluster defined within a tax- onomy. Sometimes called “cell type” here for simplicity
- Cell set accession ID: A unique ID across all tracked datasets and taxonomies. This tag labels the taxonomy and numbers each cell type. [taxonomy id][unique # within taxonomy] -CS1910121201
- Cell set alias: Any cell set descriptor (e.g., what cell types are called in a paper). It can be defined computationally based on the data, or manually based on prior knowledge or new experiments, or a combination of both:
    -ADARB2 (CGE) 
    -Inh L1-2 PAX6 CDH12
    -Chandelier
- Cell set label: An ID unique within a single taxonomy that contains a biological term that ideally is included in a relevant cell ontology. In some cases, it may be redundant with accession ID. [Canonical cell type][unique # within canonical cell type] -Neuron 12 
- Cell set preferred alias:  Analogous to “gene symbol”. At most one biologically- driven name for linking matching cell sets across taxonomies and with a reference taxonomy. 
    -L2/3 IT 4
    -Pvalb 3 
    -Microglia 2


It is necessary for this schema to be easy to understand and applicable to novel and published datasets.

Three examples are presented to illustrate how this nomenclature schema can be applied to published datasets using single nucleus RNA-seq, single cell RNA-seq, and electrophysiology.

As a whole, this workflow provides a general outline for versioned cell type classification that could be specialized as needed for communities studying different organ systems and that provides a starting point for design of future cell taxonomy and nomenclature databases.

In a reference taxonomy, the preferred alias is designed to allow tracking of the same cell types across multiple taxonomies defined using data from multiple modalities. For this reason, it should match (directly map to) cell types defined in the relevant ontology (i.e., Cell Ontology (Diehl et al., 2016) or Neuron Phenotype Ontology (Gillespie et al., 2019)), _at the highest level of resolution possible_.


An attempt has been made to extend the cell ontology to a higher resolution in human MTG with a provisional cell ontology.

The proposed schema can readily represent the matching of Chandelier cells (and other cell types) between the single-species transcriptomics datasets with the integrated transcriptomics reference and can likewise link with taxonomies from other modalities using preferred alias tags.

What should be put in the “preferred alias” identifier of provisional cell types?

Ontology curation would allow users to link data-derived cell sets to common usage terms derived from prior knowledge and that connect directly with the well-annotated ontology tools that are available for many broader cell types (e.g., the Cell Ontology)

a recent paper (Yao et al., 2020b) presents a static browser of more than 370 transcriptomically-defined cell types in mouse hippocampus and cortex, with many of
the features mentioned above, for a single taxonomy: https://taxonomy.shinyapps.io/ctx_hip_browser/.

More ambitious, the Human Cell Atlas (HCA) has proposed a tool to facilitate convergence on common cell type nomenclature through a combination of automated suggestions and other strategies, and that could potentially address some of these use cases directly. This Cell Annotation Platform is a centralized ecosystem for defining cell types and states for the Human Cell Atlas, and beyond, and is available on GitHub (https://github.com/hms-dbmi/cap-example). Whatever strategy is chosen could directly access and improve upon the proposed nomenclature convention


### Comments

## Title
    \cite{citation
    
### Highlights



### Comments