# Ontology-related
## Cell type discovery using single-cell transcriptomics: implications for ontological representation
    \cite{aevermann2018cell
### Highlights

The Cell Ontology (CL) is a biomedical ontology  to provide reference nomenclature for in vivo cell types in humans and major model organisms. However, the advent of high-content single-cell transcriptomics for cell type characterization has resulted in a number of challenges for their representation in the CL.

Propose a strategy for defining cell types within the CL

Initial efforts toward human neuronal cell type discovery focused on identifying broad lineages. Pollen et al. profiled 65 neuronal cells into six categories: neural progenitor cells, radial glia, newborn neurons, inhibitory interneurons and maturing neurons.

Comparing results between these studies has been challenging given the different areas and layers of cortex sampled.

Markers individually are still not specific enough to define discrete cell type classes at the level of granularity revealed by clustering of the sc/snRNAseq data. In fact, there has been surprisingly limited overlap in gene sets specific for individual cell type clusters between studies.

Villani et al. used fluorescence-activated cell sorting and scRNAseq to delineate six different dendritic cell subtypes (DC1–6) and four different monocyte subtypes.

-- A case of deductive reasoning (at least as reported) in assessment of cell type. --

For example, in the two studies focused on the identification of dendritic cell subtypes, it is unclear if the cDC1 and cDC2 subtypes identified by See et al. correspond to the DC1 and DC2 subtypes identified by Villani et al. Indeed, the only way to make this determination would be to perform a de novo comparative analysis of the transcriptional profiles from both studies.

One of the key recommendations was to establish a standard strategy for defining cell type classes that combine three essential components:
• The minimum set of necessary and sufficient marker genes selectively expressed by the cell type
• A parent cell class in the CL
• A specimen source description (anatomic structure þ species).

For one of the inhibitory interneuron cell types defined in this study, we were able to connect the distinct transcriptional profile with a previous cell type defined based on its unique cellular morphology — the Rosehip cell (14).

Colloquial name—rosehip neuron • Alternative name—KIT-expressing MTG cortical layer 1 GABAergic interneuron, human
• Definition—A human MTG cortical layer 1 GABAergic interneuron that selectively expresses KIT, NTNG1 and POU6F2 mRNAs.

--> I like this degree of rigorousness. <--

These informal textual definitions can then be converted into formal ontological definitions, represented in OWL as equivalent classes, using a set of logical axioms that combine assertions about the parent cell class (interneuron), anatomic locations of the neuron cell body (soma), functional capacity of the cell type (gamma-aminobutyric acid secretion) and marker gene expression (expresses some KIT) requirements (Fig. 3).

We are now establishing an extension ontology module containing provisional definitions for novel cell types that we and other research groups will contribute. In this way we will ensure the integrity of the CL reference, while still allowing for the rapid expansion of its content to accommodate cell types defined via these new technologies.

For biomedical ontologies to stay relevant, it will be critical for ontology developers to establish procedures for the processing and incorporation of representations derived from these data-intensive technologies into reference ontologies in a timely fashion. 


### Comments

Nice gathering of articles and conclusions regarding cell types! Tasty!

Number of cell types is way too big for a small group of dedicated curators. 

Also mentions of timely fashion. 

## ontoProc: Ontology interfaces for Bioconductor, with focus on cell type identification
\url{https://www.bioconductor.org/packages/release/bioc/vignettes/ontoProc/inst/doc/ontoProc.html
    
### Highlights

The ambitions of collaborative single cell biology will only be achieved through the coordinated efforts of many groups, to help clarify cell types and dynamics in an array of functional and environmental contexts. The use of formal ontology in this pursuit is well-motivated and research progress has already been substantial.

Bakken et al. (2017) discuss  ‘context annotations’ in the form of standardized experiment metadata about the specimen source analyzed and marker genes that serve as the most useful features in machine learning-based cell type classification models.”
Aevermann et al. (2018) describe how the FAIR principles can be implemented using statistical identification of necessary and sufficient conditions for determining cell class membership. They propose that Cell Ontology can be transformed to a broadly usable knowledgebase through the incorporation of accurate marker gene signatures for cell classes.

In this vignette, we review key concepts and tasks required to make progress in the adoption and application of ontological discipline in Bioconductor-oriented data analysis.

The Cell Ontology is a graphical data structure with carefully annotated terms as nodes and conventionally defined semantic relationships among terms serving as edges.
 
Given the wealth of material available in biological ontologies, it is somewhat surprising that formal annotation is so seldom used in practice. Barriers to more common use of ontology in data annotation include: 
(i) Non-existence of exact matching between intended term and terms available in ontologies of interest. 
(ii) The practical problem of decoding ontology identifiers. A GO tag or CL tag is excellent for programming, but it is clumsy to co-locate with the tag the associated natural language term or phrase. 
(iii) Likelihood of disagreement of suitability of terms for conditions observed at the boundaries of knowledge. 

To help cope with the first of these problems, Bioconductor’s ontologyProc package includes a function liberalMap which will search an ontology for terms lexically close to some target term or phrase.

-- still not exact matches -- 

The second problem can be addressed with more elaborate data structures for variable annotation and programming in R, and the third problem will diminish in importance as the value of ontology adoption becomes manifest in more applications.
 
The sym2CellOnto function helps find mention of given gene symbols in properties or parts of cell types.

The task of extending an ontology is partly bureaucratic in nature and depends on a collection of endorsements and updates to centralized information structures.

In order to permit experimentation with interfaces and new content that may be quite speculative, we include an approach to combining new ontology ‘terms’ of structure similar to those endorsed in Cell Ontology, to ontologyIndex-based ontology\_index instances.

Aaron Lun has produced a mapping from informal terms used in the Human Primary Cell Atlas to Cell Ontology tags. We provisionally include a copy of this mapping in ontoProc:

### Comments

Uses pbmc3k dataset as an example. Renders a very cool visualization of the Cell-Ontology relations wrt a dataset. 

Already uses a technotype definition!!! That is amazing. 

## Cells in ExperimentaL Life Sciences (CELLS-2018): capturing the knowledge of normal and diseased cells with ontologies
    \cite{sarntivijai2017cells
    
### Highlights

The Cell Ontology (CL) and Cell Line Ontology (CLO) have long been established as reference ontologies in the OBO framework. We have compiled a series of the challenges and the proposals of solutions in this CELLS (Cells in ExperimentaL Life Sciences) thematic series that cover the grounds of standing issues and the directions, which were discussed in the First International Workshop on CELLS at the the International Conference on Biomedical Ontology (ICBO).

After the discussion, we came to an agreement that both experimental and com- putational scientists in this domain would benefit from a shared consensus cell metadata model which can only be derived by community participation, discussion, and collaboration wherever possible. 

The workshop covered two main areas: (i) the extension of CL and CLO for ontological representation of cell types and cell lines in new technologies and experiments, and (ii) applications and challenges in real-world use cases which may require other ontological adaptations beyond CL and CLO. Examples of biomedical subject matters in the scope of this workshop were: (i) the relationship of cell components to cell types and other biomedical entities and how that may impact health and disease biology (e.g. how the subcellular component composition of certain cancer cell types reflects the prognosis and affects the pro- gression of that cancer), (ii) in vitro - in vivo relationships between primary cell samples and related immortal cell lines, and (iii) modeling of cell responses and fast-growing cell-related data generated by new technologies (e.g. novel cell types identified via interpretation of CyTOF and single cell RNA-Seq data)

New knowledge requires new vo- cabularies, but when do we stop? Or where do we begin? When should new knowledge be qualified for a new term?

When a cell can cleanly be classified with logical axioms such as those shown in the retinal bipolar cells by Osumi-Sutherland et al. (BMC Bioinformatics, this issue), there may exist another case where it may not be as straightforward to do so as shown by Bakken et al. in their attempt to perform high-content classification of brain cells (BMC Bioinformatics, this issue)

Neither the biologist community nor biomedical ontolo- gist community alone can overcome this issue.

Bringing the bench researchers who are experts in laboratory experiments to the first ICBO- CELLS workshop has exposed them to the ontology development and considerations, and at the same time, the ontologists have gained insight to the hands-on activities at the bench

When investigating how to best represent the cell modeling with ontologies, two main solutions were proposed. Should we take the Rector normalization approach [5] in building design patterns for the classification of new cell types with granular details of cell attributes such as those exemplified in bipolar neuronal cells (Osumi-Sutherland, this issue)? Or should we extend the ontology by directly asserting the new classes with axiom declarations to sup- plement the details of the new class attributes when exam- ining cells with detailed surface marker proteins? (Bekken et al., this issue).


As shown in the few scenarios here, it is clear without a doubt that experimental cell standardized nomenclature is a very important driving mechanism to move high- content single-cell research forward.

Probabilities cannot be asserted directly into the ontology, but rather using other modeling techniques such as nanopublications [8] or OBAN [9] to avoid logical errors when reasoning with the ontology.

### Comments

## Cells in ExperimentaL Life Sciences (CELLS- 2018): capturing the knowledge of normal and diseased cells with ontologies
    \cite{sarntivijai2019cells
    
### Highlights

The 2nd International Workshop on Cells in ExperimentaL Life Science (CELLS-2018) focused on two themes of knowledge representation, for newly-discovered cell types and for cells in disease states.

The general discussion session focused on the ontology design patterns in representing newly-discovered cell types and cells in disease states.

Normalization of experimental cell data produced in different laboratory settings is difficult due to the non-synchronous nature of multiple laboratories working on similar questions on the same timeline

The CELLS-2017 workshop included 7 oral presentations, and eventually five full-length articles [5–9]

The half-day CELLS-2018 workshop included five oral presentations and a general discussion session. Four of the proceedings papers, corresponding to four oral presentations, have been extended, further reviewed, and are published in this special issue for the CELLS-2018 workshop in BMC Bioinformatics.

In this workshop, two new ontologies were reported. Serra et al. presented the development of the Cancer Cell Ontology (CCL) that represents cancer cell types.

The other reported new ontology is the Ontology for Stem Cell Investigations (OSCI) [12]

To support worldwide interoperable cell line infor- mation representation and usage, Pan et al. reported the development of CLO-NICR ontology that represents these cell lines using the semantic framework of CLO.

The CLO-NICR development was one of the research efforts from the OntoChina (http://www.ontochina.org)[15], a new initia- tive supporting an ontology developer community program in China to promote collaborative ontology development and applications around the world
In a CELLS-2018 presentation, Vita et al. applied ontologies to connect cell population descriptions and gating defi- nitions [16]. In their study, ontologies were used to cross-compare cell types and marker patterns in the ImmPort Immunology Database and Analysis Portal [17]. 

Mohamed Keshk of Richard H. Scheuermann’s group presented a provisional cell type ontology that represents newly identi- fied novel cell types experimentally identified using single cell RNA sequencing (scRNAseq), an extension of the work Dr. Scheuermann’s group presented in the CELLS-2017 workshop [5].

For example, what is the difference between normal and diseased cells, and how to onto- logically represent the transformation from a normal cell to diseased cell.

The community discovers new cell types based on subdividing known cell populations with additional markers. As we move beyond marker definition of cell types developed via flow cytometry to defining cell types via single cell RNA sequencing or proteomics, a cell type could potentially be defined by a combinations of hun- dreds of marker genes or proteins. Should

As reflected by the work of Dr. Scheur- mann’s group and others [19], we might therefore con- sider what constitutes the minimal combination set of markers that can uniquely define a new cell type

Given the diver- sity of the experimental approaches for studying cells, we need to be careful in how we represent experimental results in the form of ontological definitions of cell types

A variety of recent studies have discovered many new cell types, which sometimes show different characteristics in normal or disease states [22–26].

We anticipate promising research progress in the cell-related ontology field in the years to come.


### Comments


## Reporting and connecting cell type names and gating definitions through ontologies
    [@doi:10.1186/s12859-019-2725-5]
    
### Highlights
Human immunology studies often rely on the isolation and quantification of cell populations from an input sample based on flow cytometry and related techniques. Such techniques classify cells into populations based on the detection of a pattern of markers. The description of the cell populations targeted in such experiments typically have two complementary components: the description of the cell type targeted (e.g. ‘T cells’), and the description of the marker pattern utilized (e.g. CD14−,CD3+).

We here describe our attempts to use ontologies to cross-compare cell types and marker patterns (also referred to as gating definitions). 

We used a large set of such gating definitions and corresponding cell types submitted by different investigators into ImmPort, a central database for immunology studies, to examine the ability to parse gating definitions using terms from the Protein Ontology (PRO) and cell type descriptions, using the Cell Ontology (CL). We then used logical axioms from CL to detect discrepancies between the two.

We also suggest a number of new terms to describe gating definitions in flow cytometry that are not based on molecular markers captured in PRO, but on forward- and side-scatter of light during data acquisition, which is more appropriate to capture in the Ontology for Biomedical Investigations (OBI). Finally, our approach results in suggestions on what logical axioms and new cell types could be considered for addition to the Cell Ontology.

An initial examination of the data submitted to Imm-
Port revealed that, in the absence of precise standards.

We describe here our attempts to standardize the “cell population” field, by mapping it to terms in the Cell Ontology

As a result of this work, we propose a new convention on how these two fields should be populated that allows one to in- terpret them computationally, query across them, and to utilize logical axioms in the Cell Ontology to check for consistency between them.

“singlet/ CD14−/CD3+” is a gating definition from a center using slashes as separators which consists of three gates: “sing- let”, “CD14−”, and “CD3+”. 

Our purposes to recognize PRO as the central arbiter and translator of protein names.

While the synonyms in Table 1 were considered not problematic, we also found a number of labels that could not be mapped precisely to individual proteins in PRO. A common case were protein complexes: Investigators often refer to “CD8”, which is a complex of a CD8 alpha

Ideally, all of these protein complexes would be included in PRO. This work should be co- ordinated with the Cell Ontology, to ensure it matches the logical cell type definitions.

The use of cocktails of markers to include or exclude lineages of cells (e.g. ‘Lin-’) is hard to represent as different cocktails exist.

Several gating definitions we encountered for flow cy- tometry data do not refer to a specific marker, but ra- ther utilize forward- and side-scatter of detector light to provide size estimates of the object passing by the detector.

Such definitions of cell types based on their characteristics in a flow cytometer are best captured in the Ontology of Biomedical Investiga- tions (OBI) [9], and we are in the process of submitting them there.

The guide- line to use CL terms for submission of cell populations to ImmPort was found to be insufficient, as most cell populations gated for are further subsets or activation states of CL types


Thus, we revised our approach and developed a future tentative guideline whereby the data provider specifies the parent cell type in CL and adds additional markers or cell subtypes after an ‘&’ symbol. For example, “BDCA3\_pmDC” was parsed to ‘plasmacy- toid dendritic cell’ & “BDCA3+”.

In order to detect such discrepancies, we implemented
a prototype validator, shown in Fig. 1, which can be accessed through https://github.com/jamesaoverton/cell- name-and-marker-validator. The proposed workflow is that users would input “Population name” and “Gating definitions” fields and the validator will check both for
validity and cross-check for consistency

We then wrote a second Python program to demonstrate our vi- sion of the standard terminology going forward as an interactive web page. This tool reads labels and logical definitions from the Cell Ontology, PRO short labels from the Protein Ontology, and our manual list of gate names. The user is asked to enter a cell population name from the Cell Ontology and a set of gates. The tool then displays the logical definition for the cell type and the gating strategy, and highlights any conflicts between the two.

Here we demonstrated that a vast number of gating defi- nitions, but not cell population names, could be mapped to terms in ontologies, allowing us to semantically ex- press them and detect potential clashes.

Some gate names occurred nearly 2000 times, while others only occurred once

The relationships between “Population name” and “Gating definition” are not abso-
lute, for example if the input sample type is PBMC ver- sus splenocytes (“Population name”) with the same sorting definition (“Gating definition”) the outcome will be different cell types. This means that the validation cannot be expected to catch every potential error and fu- ture refinement may be possible after larger datasets are generated using these criteria. We also recognize that our approach is human-centric due

Lastly, and signifi- cantly, implementation of this validation into the Imm- Port submission templates is required if it is to be practical and easy for the HIPC community to utilize.
Abbreviations

Table 2: 
Markers that could not be precisely mapped to individual PRO protein chains

### Comments

Seems that they a aim at a very advanced very level of usage before we lay a bit more of the foundations. 

Manual curation will not happen for all proteins of all organisms. 

https://github.com/jamesaoverton/experimental-cell-ontology


## An ontology for cell types
[@doi:10.1186/gb-2005-6-2-r21]

### Highlights
the annotation of  phenotypes is hampered by a lack of structured information describing a variety of other biological objects, including cell types.

To address these needs, we have developed an ontology that describes the cell types of the major model organisms, both animal and plant.

Its use will allow a biologist to query a sin- gle database with such questions as: list all of the cell types in
mouse that express the Notch gene and all of the cell types in Drosophila and Caenorhabditis elegans that express the closest homolog of this gene.

The first design decision was whether we should attempt to integrate cell types from all phyla within a single ontology or build independent ontologies for different taxonomic groups.

Despite these problems in building an 'integrated' cell-type ontology, the advantages, were we to succeed, outweigh them, and we have therefore taken this approach to develop a single ontology that inte- grates cell types from different phyla.

ontology appears as a complex hierarchy (technically known as a directed acyclic graph, or DAG) where a given term (or concept) may not only have several children, but also several parents. The parent and child terms are connected to each other by is_a and develops_from relationships.

it is far from obvious that vertebrate and inverte- brate pigment cells are homologous and these concepts are therefore described as pigment_cell_(sensu_Vertebrata) and pigment_cell_(sensu_Nematoda_and_Protostoma, respectively.

The two top-level nodes of the Cell Ontology are cell_in_vivo and experimentally_modified_cell.

The organism-independent classification of cells follows several different criteria that include: 'function' (for example, electrically_excitable_cell, secretory_cell, photosynthetic_cell), histology (for example, epthelial_cell, mesenchyme_cell), lineage (for example, ectodermal_cell, endodermal_cell) and ploidy (for example, haploid_cell, polyploid_cell).

Kupffer cells are specialized vertebrate macro- phages of the reticuloendothelial system. They function to fil- ter small foreign particles (including bacteria) and old reticulocytes from the blood. In the Cell Ontology they are to be found by their function (they are a type of defensive_cell), by their lineage (they are derived from a mesodermal_cell derived from a hematopoietic_stem_cell, itself a type of stem_cell), by their morphology (they are a type of circulating_cell) and by their organism (they are a type of animal_cell).


It is designed to be useful in the sense that a researcher should be able to find, in a rapid and intuitive way, any cell type in any of the major model organisms and, having found it, learn a considerable amount about that cell type and its relationships to other biological objects.

Cell-type unique identifiers (ID) that can be incorporated into any database holding cell-type-associated knowledge.

While we expect such bio- informatics applications to be its immediate use, we hope that, in the longer term, all biologists will find the ontology useful

Cells are the biological units with which tissues and organs and organ systems are built. A rich and explicit description of cell types across phyla that are adapted by bio- logical databases will help facilitate this transition.

These cell types have been collated from our own knowledge, from major textbooks (for example [20-22])

20. Williams PL, (Ed): Gray's Anatomy 38th edition. Edinburgh: Churchill Livingstone; 1996.
21. Alberts B, Johnson A, Lewis J, Raff M, Roberts K, Walter P: The Molec- ular Biology of the Cell 4th edition. New York: Garland; 2002.
22. Esau K: The Anatomy of Seed Plants 2nd edition. New York: John Wiley; 1977

### Comments

The architecture of CL has changed over time. 

The CL had originally a _learning_ goal (among others) : "learn a considerable amount" . This explains a "bias" towards didatic definitions of cell types. 

## Logical Development of the Cell Ontology
[@doi:10.1186/1471-2105-12-6]

### Highlights
Here we report onthe generation of computable definitions for the hematopoietic cell types in the CL.

Computable definitions for over 340 CL classes have been created using a genus-differentia approach

The implementation of this process exposed areas in the ontology where new cell type classes were needed to accommodate species-specific expression of cellular markers.


Use of computable definitions enhances the development of the CL and supports the interoperability of OBO ontologies.

First developed in 2004 [3], the CL is composed of
about 1000 classes connected via is_a and develops_from relationships.


The initial classification was done manually and not in accordance with commonly accepted ontology engineering princi- ples such as normalization and modularity [4]. The resulting ontology contained numerous mistakes and omissions, and proved difficult to maintain and extend.

computable definitions (sometimes called “logical definitions” or “cross-products”) are treated as equivalence relationships between the defined class and
some conjunction of classes. For example, the class “nucleate erythrocyte” can be defined as equivalent to the class of things that are both erythrocytes and that
have a nucleus as a part.


Our computable defini- tions all follow a genus-differentia structure, in which a class is defined by refining an existing more general class (the genus - in the above example this would be “erythrocyte”) and one or more differentiating character- istics (the differentiae - in the above example this is the has_part relationship to nucleus).

They defined dendritic cell types using cell surface expression of proteins as a major axis of dif- ferentia. Thus, a “CD8_alpha-negative, CD11b-negative dendritic cell” is a “conventional dendritic cell” (the genus) that “lacks_plasma_membrane_part CD8_alpha.


By integrating Masci et al.’s approach with our recent reorganization of the hematopoietic branch of the CL
[13], we have created a new sub-ontology called “Hemo_CL

By having computable definitions for the vast majority of classes, new relationships can be inferred between classes by the use of automated reasoners.

278 cell type classes defined according to this template have been generated, reflecting the common use of flow cytometry to distinguish hematopoietic cells based on cell surface markers.

Some of our definitions also made use of negative criteria - for example, an alveolar macrophage is partially defined as a tissue-resident macrophage that lacks che- mokine receptor CX3CR1 on its plasma membrane

Pro-B cells, for example, are described as being in the “early stages of recombination of B cell receptor genes”,soa computable definition using “participates_in immuno- globulin V(D)J recombination (GO:0033152)” is appro- priate.

the only similarity in cell surface markers between mouse and human HSC is that neither express cell surface pro- teins associated with fully differentiated immune cells (CD3epsilon, CD19, etc), a characteristic researchers call lineage-negative (lin-). Human HSC express CD34 and lack expression of CD38 on their cell surface whereas mouse HSC express other proteins, Kit and Spa-1, on their cell surface.

New cell types have been added to represent hemato- poietic progenitor cell types that are currently an intense focus of research

The transitive nature of the develops_from relationship allowed the reasoner to infer develops_from relationships between classes that are separated by multiple levels in the asserted is_a hierarchy

Errors that contradict biological knowledge were checked by using disjointness constraints.

Computable definitions can reveal ambiguities in how a cell type is defined by a curator. One example is the cell types “natural T regulatory cell” (nTregs) and “induced regulatory cell” (iTregs). Because the cell types have the same is_a parent and share two cell surface markers, the reasoner inferred the former to be a subtype of the latter in an earlier version of the ontology, violating a disjointness constraint existing between the two classes.

Some ambiguities resulted from how a cell type is defined in the scientific literature. The automated rea- soning for the “mature NKTcell” provided an interest- ing example. Reasoning over the ontology failed to infer an is_a relationship between “NK T cell” and the “natural killer cell” despite “NK T cell” having the synonym “natural killer T cell” (not shown). Investigation of the literature revealed that the name “natural killer T cell” is controversial; the origin of the name “NK T” cell refers to cell surface expression of the NK1.1 protein and was not intended to imply a direct link to natural killer cells [21]. However some researchers feel there is enough functional overlap between “NK-” and “NK T-” cell types to justify the “natural killer T cell” moniker [21]. In Hemo_CL, the synonym is left in place as it reflects common usage, but NK T cell types and natural killer cell types are kept in separate branches of the ontology as inferred by the reasoner.


Computable definitions for hematopoietic cell types have importance beyond ontology maintenance and integration for the wider scientific community. 

We believe hypothesis- generation represents a future use of well-structured biomedical ontologies that have computable definitions.


We generated new hematopoietic cell types to repre- sent entities described in current research and added computable definitions for all hematopoietic cell types using classes from other OBO library ontologies.

(note that this ontology has been integrated into the
main CL ontology, but a record of the Hemo_CL subset as described in this paper will remain in the source con- trol system. The revision number described in this paper is hemo_CL.obo revision 1.47).



### Comments

A "rigorous definition" is somewhat a looser version of a "computable definition".  
# Template 

## Title
[citation]
### Highlights



### Comments