# Other



## Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain
    \cite{[@doi:10.7554/eLife.38619]
    
### Highlights
200 genetically identified cell populations, and two gene classes contributing distinctly to neuronal diversity.

Cells from micro-dissected brain regions may contain more than one atomic cell type, we refer to these as genetically- and anatomically-identified cell populations (GACPs).
<-- "atomic cell type", does this concept even make sense? -->

Pan-neuronal genes (genes expressed commonly in all neuronal populations but expressed at lower levels or not at all in nonneuronal cell populations.

Gene families to provide information about cell identities which  is not captured by ’gene-wise’ metrics like mean DEF, or by enrichment analysis.

This suggests that, similar to other tissues homeobox TFs play an important role in specifying cell types in the brain.

Alternative splicing: most branch point choices are made consistently, in an all-or-none fashion, for any given cell population

We assume that cell types are organized hierarchically in a tree-like fashion proceeding from major branches (e.g. ’cortical excitatory neuron’) to more specialized subtypes, with the terminal ’leaf-level’ branches comprising ’atomic’ cell types. Profiled cell populations are defined operationally by the intersection of a transgenic mouse strain (or in some cases anatomical projection target) and a brain region.

### Comments

"atomic cell type", does this concept even make sense?

Treats groups not as cell types, but as genetically- and anatomically-identified cell populations (GACPs).

Explicit assumption of hierarchy and atomic cell types!

## Terminal Selectors of Neuronal Identity
    [@doi:10.1016/bs.ctdb.2015.12.007]
    
### Highlights

Terminal identity features of a mature neuron are often coregulated by so-called terminal selector transcription factors.

Emphasize core features of this concept in the C. elegans system such as coregulation of terminal effector batteries, combinatorial control mechanisms, and the coupling of initiation and maintenance of neuronal identity.

Molecular characterization of these behavioral mutants showed that many of them affect tran- scription factors that turned out to act at the terminal step of differentiation of postmitotic neurons.

These molecular markers, built by the community over the past 20 years (curated at www.wormbase.org), monitor the neuron-type specific expression of hundreds of genes that define terminal properties of a neuron type.

Given the neuron identity-defining properties, I proposed to call these transcription factors “terminal selectors.This terminology is an extension of the selector gene concept, originally proposed by Garcia-Bellido (1975).

Identity-defining terminal effector genes are coregulated.

Terminal selectors control regulatory subroutines

Every fully differentiated neuron type is uniquely defined by the expression of a unique combination  of “nuts and bolts,” “terminal effector” genes that code for proteins that define the functional features of a mature neuron type.

Even in a nervous system as comparatively simple as C. elegans, only very few terminal effector genes are exclusively expressed in a single neuron type.

A key defining feature of terminal selectors is that they are are continuously expressed throughout the life of the neuron to maintain the differentiated state of a neuron.

Terminal selectors are combinations of cooperating factors.

There may be a spectrum of distinct types of cooperation between terminal selectors, depending on cis-regulatory architecture of individual effector genes.

There are a number of key implications of the combinatorial activity of terminal selectors.

Terminal selectors control the expression of many if not all  effector genes expressed in a neuron-type-specific manner and which define the identity of a neuron type. 

However, terminal selectors do not control the expression of all genes in a terminally differentiated neuron type.

At the time of writing the original essay on the terminal selector concept, only a handful of examples of such selector genes has been described.

The purpose of this chapter has been to provide a description of the general features of gene regulatory programs that operate in terminally differentiated neuron types throughout the C. elegans nervous system

### Comments

118 anatomically defined neuron types 

## Construction of a human cell landscape at single-cell level
    \cite{citation
    
### Highlights

Master TFs work in discrete modules to specify major human cell types such as neuron, erythroid cell and cainar cell. 

Mapping of human cell types by  scHCL. Match to reference transcriptomes by correlation. 

Integrated data to the Mouse Cell Atlas.

We propose a landscape model that is conserved in mammal: stem cells are transcriptionally "indistinct" and "stochastic" and differentiated cells are transcriptionally "distinct" and "stable"; the wiring of the regulons in the genome pre determines the terminal steady cellular states.
### Comments

## The Definition of Cell Type
    [@doi:10.1161/01.RES.84.10.1234]
    
### Highlights
What do we mean by “cell type?” 

We know that cultured smooth muscle cells obtained in different ways have different stable phenotypes. These results suggested that the cultures somehow represent a model for the cells forming the neointima in vivo.


One implication of definitive analyses of gene expression should be a redefinition ofwhat we mean by cell type. Currently “cell type” is defined, in most cases, either by morphology or by a small number of genes that have been identified with that tissue.

In summary, biology is changing. We have the tools to explore hypothesis-based studies in increasingly minute detail. At the same time, systematic exploration will provide us with more pathways to be defined. As such studies proceed, the functional map will get more and more complex. The challenge will be to discriminate critical mechanistic steps from less important details.


### Comments
"These studies, taken together, illustrate a major change in how we define cell type, a change that is about to be accelerated by the power of systematic genomics" --> Initial description of how transcriptomics would change our view of cell type diversity


Addresses a bit of the difference between native cell types and cultured cell types.


# Template 

## The Human Cell Atlas: from vision to reality
[@doi:10.1038/550451a]
### Highlights

As an ambitious project to map all the cells in the human body gets officially under way, Aviv Regev, Sarah Teichmann and colleagues outline some key challenges.

For more details on how the atlas will be built as an open global resource, see the white paper2 posted on the Human Cell Atlas website.

In cellular taxonomy, it might enable the discovery and identification of cell types and molecular markers or signatures (a collection of genes, say, that characterize a specific cell type).

“To have maximum impact, the Human Cell Atlas must be an open resource, on multiple levels.”

The first draft of the atlas will profile cells’ molecular and spatial characteristics, capturing only those cell types that occur above a pre-specified rarity — ones that make up more than 1% of a sample, say.

A similar step-wise strategy was deployed in the Human Genome Project; even a partially assembled genome proved immediately useful to researchers, and human genetic variation in health and disease was tackled over several years after the full genome was sequenced. 

As much as possible, the atlas must reflect the diversity of humans and human experience.

Descriptors such as ‘cell type’ and ‘cell state’ can be difficult to define at the moment. An integrative, systematic effort by many teams of scientists working together and bringing different expertise to the problem could dramatically sharpen our terminology, and revolutionize the way we see our cells, tissues and organs. We invite you to join the effort.

### Comments

# Template 

## CELLPEDIA: a repository for human cell information for cell studies and differentiation analyses
[@doi:10.1093/database/bar046]
### Highlights

CELLPEDIA features 
- (i) its own cell classification scheme, in which whole human cells are classified by their physical locations in addition to conventional taxonomy; and 
- (ii) cell differentiation pathways compiled from biomedical textbooks and journal papers. Currently, human differentiated cells and stem cells are classified into 2260 and 66 cell taxonomy keys, respectively, from which 934 parent–child relationships reported in cell differentiation or transdiffer- entiation pathways are retrievable.

Many histology and molecular biology textbooks state that there are ?200 types of cells in the adult human body (2). However, this classical observa- tion is mainly based on the morphological or histological perspective, and no cell classification system based on gene expression levels has yet been systematically established.

Inspired by the importance of cell studies early on, we developed a gene expression profile data search system called ‘CellMontage’ in 2004, where we also provided a draft version of a cell catalogue (13).

The latest version of CELLPEDIA consists of two major cell categories: ‘differentiated cell’ and ‘stem cell’. The ‘differ- entiated cell’ category contains information of fully differ- entiated or mature cells that represent specific functions or characteristics, whereas the ‘stem cell’ category contains information of undifferentiated or immature cells, such as progenitor or multipotent stem cells.

In typical textbooks, cells with equivalent or similar func- tions are given the same name even if they exist in different physical locations. For instance, ‘goblet cells’ are found in the nose, lung and small and large intestines.

CELLPEDIA defines its own enhanced cell taxonomy for physically distinct cells, establishing a hierarchical structure of up to eight levels from organs (or tissues) to cells. It presents a combination of both conventional cell taxonomy and their physical locations (Figure 1a).

To avoid con- fusion, we have added a ‘Population’ category in the gene expression data table to provide information of the data resource. ‘Heterogeneous’ means that the gene expression data are measured from a heterogeneous cell population

--> Which population is not heterogeneous??
### Comments
 It became extended to https://stemcellinformatics.org/
# Template 

## Title
[@doi:]
### Highlights



### Comments