# Other



## Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain
    \cite{sugino2019mapping
    
### Highlights
200 genetically identified cell populations, and two gene classes contributing distinctly to neuronal diversity.

Cells from micro-dissected brain regions may contain more than one atomic cell type, we refer to these as genetically- and anatomically-identified cell populations (GACPs).
<-- "atomic cell type", does this concept even make sense? -->

Pan-neuronal genes (genes expressed commonly in all neuronal populations but expressed at lower levels or not at all in nonneuronal cell populations.

Gene families to provide information about cell identities which  is not captured by ’gene-wise’ metrics like mean DEF, or by enrichment analysis.

This suggests that, similar to other tissues homeobox TFs play an important role in specifying cell types in the brain.

Alternative splicing: most branch point choices are made consistently, in an all-or-none fashion, for any given cell population

We assume that cell types are organized hierarchically in a tree-like fashion proceeding from major branches (e.g. ’cortical excitatory neuron’) to more specialized subtypes, with the terminal ’leaf-level’ branches comprising ’atomic’ cell types. Profiled cell populations are defined operationally by the intersection of a transgenic mouse strain (or in some cases anatomical projection target) and a brain region.

### Comments

"atomic cell type", does this concept even make sense?

Treats groups not as cell types, but as genetically- and anatomically-identified cell populations (GACPs).

Explicit assumption of hierarchy and atomic cell types!

## Terminal Selectors of Neuronal Identity
    \cite{hobert2016terminal
    
### Highlights

Terminal identity features of a mature neuron are often coregulated by so-called terminal selector transcription factors.

Emphasize core features of this concept in the C. elegans system such as coregulation of terminal effector batteries, combinatorial control mechanisms, and the coupling of initiation and maintenance of neuronal identity.

Molecular characterization of these behavioral mutants showed that many of them affect tran- scription factors that turned out to act at the terminal step of differentiation of postmitotic neurons.

These molecular markers, built by the community over the past 20 years (curated at www.wormbase.org), monitor the neuron-type specific expression of hundreds of genes that define terminal properties of a neuron type.

Given the neuron identity-defining properties, I proposed to call these transcription factors “terminal selectors.This terminology is an extension of the selector gene concept, originally proposed by Garcia-Bellido (1975).

Identity-defining terminal effector genes are coregulated.

Terminal selectors control regulatory subroutines

Every fully differentiated neuron type is uniquely defined by the expression of a unique combination  of “nuts and bolts,” “terminal effector” genes that code for proteins that define the functional features of a mature neuron type.

Even in a nervous system as comparatively simple as C. elegans, only very few terminal effector genes are exclusively expressed in a single neuron type.

A key defining feature of terminal selectors is that they are are continuously expressed throughout the life of the neuron to maintain the differentiated state of a neuron.

Terminal selectors are combinations of cooperating factors.

There may be a spectrum of distinct types of cooperation between terminal selectors, depending on cis-regulatory architecture of individual effector genes.

There are a number of key implications of the combinatorial activity of terminal selectors.

Terminal selectors control the expression of many if not all  effector genes expressed in a neuron-type-specific manner and which define the identity of a neuron type. 

However, terminal selectors do not control the expression of all genes in a terminally differentiated neuron type.

At the time of writing the original essay on the terminal selector concept, only a handful of examples of such selector genes has been described.

The purpose of this chapter has been to provide a description of the general features of gene regulatory programs that operate in terminally differentiated neuron types throughout the C. elegans nervous system

### Comments

118 anatomically defined neuron types 

## Construction of a human cell landscape at single-cell level
    \cite{citation
    
### Highlights

Master TFs work in discrete modules to specify major human cell types such as neuron, erythroid cell and cainar cell. 

Mapping of human cell types by  scHCL. Match to reference transcriptomes by correlation. 

Integrated data to the Mouse Cell Atlas.

We propose a landscape model that is conserved in mammal: stem cells are transcriptionally "indistinct" and "stochastic" and differentiated cells are transcriptionally "distinct" and "stable"; the wiring of the regulons in the genome pre determines the terminal steady cellular states.
### Comments

## The Definition of Cell Type
    \cite{schwartz1999definition
    
### Highlights
What do we mean by “cell type?” 

We know that cultured smooth muscle cells obtained in different ways have different stable phenotypes. These results suggested that the cultures somehow represent a model for the cells forming the neointima in vivo.


One implication of definitive analyses of gene expression should be a redefinition ofwhat we mean by cell type. Currently “cell type” is defined, in most cases, either by morphology or by a small number of genes that have been identified with that tissue.

In summary, biology is changing. We have the tools to explore hypothesis-based studies in increasingly minute detail. At the same time, systematic exploration will provide us with more pathways to be defined. As such studies proceed, the functional map will get more and more complex. The challenge will be to discriminate critical mechanistic steps from less important details.


### Comments
"These studies, taken together, illustrate a major change in how we define cell type, a change that is about to be accelerated by the power of systematic genomics" --> Initial description of how transcriptomics would change our view of cell type diversity


Addresses a bit of the difference between native cell types and cultured cell types.

