
# Computational single-cell research
## Searching large-scale scRNA-seq databases via unbiased cell embedding with Cell BLAST
    \cite{cao2020searching
    
### Highlights
Herein, we present Cell BLAST, cell-querying method built on a neural network-based generative model and a cell-to-cell similarity metric.

Intuitively, low-dimensional embeddings of cells from the same cell type should reconstruct each other better than cells from different cell types. 

Our generative model combined with posterior-based similarity metric enables Cell BLAST to model the continuous spectrum of cell states more accurately.

Constructing a large-scale well-curated reference database.

A comprehensive and well-curated reference database is crucial for the practical application of Cell BLAST. 

Empirical case study shows that both the incorporation of hit similarity and ontology structure lead to improved Cell Ontology assignment.

Metadata were further manually curated by adding additional descriptions in the paper to acquire the most detailed information of each cell.

We unified raw cell type annotation by Cell Ontology. Closest Cell Ontology terms were manually assigned based on the Cell Ontology description and context of the study.

BLAST2CO. BLAST2CO can perform Cell Ontology (CL) inference for query cells as long as the reference datasets are annotated with Cell Ontology. For each CL existent in the hit cells, BLAST2CO calculates its confidence score by summing up the similarity of all hit cells belonging to the particular CL. 
The confidence scores are assigned to corresponding nodes in the CL graph, and then propagated to parent nodes along the graph.

If more than one CLs match the above conditions, the query cell is labeled as “ambiguous”. If no CL matches the above conditions, the query cell is labeled as “rejected.”

### Comments

"cell type resolution" is an interesting concept. 

Phrases like "All five ionocytes predicted as club cells by Cell BLAST are also agreed on by scmap." cast a shadow of doubt on the original annotations. 
"continuous spectrum of cell states more accurately."  cell states are very explictly assumed to lie on a continuous spectrum
Annotation of cell types with Cell Ontology is strictly wrong (albeit being better than no annotatino at all)
Annotates each single cell with a ranking of "terminal" cell types. 

The ppropagation of annotations is very useful. 

## CellFishing.jl: an ultrafast and scalable cell search method for single-cell RNA sequencing
    [@doi:10.1186/s13059-019-1639-x]
    
### Highlights
We propose CellFishing.jl, a new method for searching atlas-scale datasets for similar cells.

This trend of sequencing higher numbers of cells is expected to continue until a complete list of cell types is generated.


Cell type estimation accomplished by matching query cells with similar cells found in anno- tated data sets provides important information concerning the query cells, relying on a single similarity score may result in overlooking significant differences in their gene expressions.

For example, the developmental stages of the hematopoietic lineage from stem cells to completely differentiated cells are often characterized by the expression level of few marker genes. 
Additionally, using scRNA-seq, Park et al. revealed that genes related to Mendelian disease are differentially expressed in specific cell types.
These facts indicate that mutually similar cells of the same type but under different conditions can be further distin- guished by noting differentially expressed genes (DEGs) between these cells

We next confirmed that the hashed expression profiles preserve the original differences among cell types by visualizing low-dimensional embedding of the data.

CellFishing.jl evidently includes minor cell types, although the scores are relatively unsta- ble for those cell types. For example, while Baron2016 contains only 18 Schwann cells and seven T cells, CellFishing.jl found these cell types with high consistency.

Comparing transcriptome expressions across different species provides important information on the function of unknown cell types. Since the Baron2016 data set includes cells derived from human and mouse, we attempted to match cells between both species. A total of 12,413 one-to-one gene pairs were included

### Comments

What is a "complete list of cell types" !?


## Probabilistic gene expression signatures identify cell-types from single cell RNA-seq data
    \cite{grabski2020probabilistic
    
### Highlights

Here we present a computationally light statistical approach, based on Naive Bayes, that leverages public datasets to combine information across thousands of genes and probabilistically assign cell-type identity.

This improved granularity permits the identification or discovery of distinct populations of cell-types within the tissues under study. To effectively accomplish this, it is important to classify cells reliably into known cell-types, in particular cells that are present in many tissues such as immune system cells.

For instance, in CD4 cells, the markers had non-zero counts in anywhere from 0\% to 38\% of the cells. As a result, many marker genes that have been established as reliable indicators of cell-type on the bulk level are no longer adequate on the single cell level.

Naive Bayes [12], that defines a conditional distribution for each target cell by estimating a cell-type-specific expression rate for each gene

"These findings suggest that the identity assigned to a given cell depends on the size of the dataset, in a way that is not representative of the actual cell-types in the data"


Specifically, we selected marker genes for CD4 (IL7R, CD4, CTLA4, FOXP3, IL2RA, PTPRC), CD14 (CD14, LYZ, FCGR3A, CD68, S100A12), CD8 (CD8A, CRTAM, NCR3, CD3D), and NK cells (GNLY, NKG7, PRR5L, S1PR5, NCAM1) that have been curated and well-established in the literature as discriminating among cell types 

We explicitly include the total number of observed transcripts Ni to account for the varying coverage across cells [24]. Note that the vector (λ1k, . . . , λJk)? can be considered the expression profile that defines cell-type k.

this approach allows the certainty of the classification to be directly quantified. For example, if a target cell has a relatively low probability of belonging to any of the reference cell-types, this might indicate contamination or potentially a novel cell-type population.

We explicitly model the cell-type-specific probability distributions of these genes with a hierarchical approach that increases power in the presence of sparsity.


### Comments

Interesting. We want to study the cell types themselves, not the cells per se. 


## CHETAH: a selective, hierarchical cell type identification method for single-cell RNA sequencing
    [@doi:10.1093/nar/gkz543]
    
### Highlights
CHETAH (CHaracterization of cEll Types Aided by Hierarchical classification) is an accurate cell type identification algorithm.

Single cell RNA sequencing has paved the way for discovering previously uncharacterized cell types and this application too would greatly benefit from efficient identification of known cell types prior to focusing on new types

Analyses are becoming more complex given the fast-expanding catalogue of defined cell types (15).

For example, if the evidence for a very specific cell type assignment such as effector CD8 T-cell is not strong enough, assignment such as T cell may  be useful.

The correct T-cell subtype may be not part of the reference dataset, or even that there is in- sufficient read-depth for the more specific call to be made. CHETAH’s classification is a stepwise process that traverses the tree and, depending on the available evidence, ends at one of the reference cell types or halts at the unassigned or one of the intermediate types.
 
It is firmly established that hierarchical trees derived from clustering gene expression data reflect many aspects of the underlying biology (52). <-- Strong affirmation -->

For the method presented here, the definition of cell type is pragmatic and can best be described as any group of cells annotated within a reference set as belonging together.

### Comments

argues that we need to identify what we know before we can efficiently focus on new types. 

explicitly assumes one tree

explicitly defines cell type! awesome!


## R Package: Single-cell RNAseq cell cluster labelling by reference
    \url{https://www.bioconductor.org/packages/release/bioc/vignettes/celaref/inst/doc/celaref_doco.html
    
### Highlights

 The annotation of the ‘cell type’ of each cluster is performed by a domain expert biologist 
 
 A list of ‘Up’ genes are extracted for each query cluster. Then, those genes are looked up in the ranking of genes in each reference cell cluster. 
 
 There can be none, one or multiple reference group similarities for the query group.
 
 
### Comments

Uses a lot of p values, but null hypothesis are not clear. The word "null" is not mentioned once.


## PanglaoDB: a web server for exploration of mouse and human single-cell RNA sequencing data
    \cite{franzen2019panglaodb
    
### Highlights

A community-curated cell-type marker compendium.

Deposited data often remain difficult to access as it requires significant pre-processing to become useful for regular analysis.

Our database provides a manually curated list of cell-type markers that can be incorporated into novel algorithms for inference of cell types.

The database of cell type markers was compiled by
manual curation of thousands of published articles and abstracts, and by querying internet search engines with strings such as ‘GENE1 is expressed in ∗ cells’.

The cell-type identity is then determined using the collection of cell-type marker genes. Some marker genes are found in multiple cell types and are less informative for cell-type inference.

In such cases it is possible to compare if the predicted cell type matches the ‘biological’ cell type. We identified 17 independent datasets with homogeneous cell-type populations, where the biological cell type has been reported.

At the time when this work was initiated, there was no published database or comprehensive list of genes that can be used for automatic cell-type assignment [CellMarker (41) came out when this manuscript was in preparation].

### Comments



## Unifying single-cell annotations based on the Cell Ontology
[@doi:10.1101/810234]
### Highlights

Here, we present OnClass, an algorithm and accompanying software for automatically classifying cells into cell types part of the controlled vocabulary that forms the Cell Ontology.
It uses the Cell Ontology graph to infer cell type relationships.

OnClass can be used not only as an annotation tool for single cell datasets but also as an algorithm to identify marker genes specific to each term of the Cell Ontology, offering the possibility of refining the Cell Ontology using a data-centric approach.

The Cell Ontology offers a controlled vocabulary for cell types and has been proposed as the basis for consistently annotating large-scale single-cell atlases.

Assigning cells to terms (i.e., cell types) in the Cell Ontology has at least three challenges:

- most of these cell type terms are not associated with marker genes.
- supervised learning approaches can be used to predict Cell Ontology terms that have curated annotations, they are unable to classify cells to unseen terms
- The Cell Ontology is not developed specifically for scRNA-seq, it likely misses new cell types and cell states and so certain cell type relationships might be inaccurate

Since OnClass annotated cells even to previously unseen Cell Ontology terms according to the annotated cells from other Cell Ontology terms, its performance greatly relied on the quality of cell type embeddings.

In the state-of-the-art single-cell integration frameworks, datasets from different technologies are aligned in the same low-dimensional space by using mutual nearest neighbors as anchors to connect them. Indeed, our method can be considered to be aligning the Cell Ontology to the gene expression matrix by using known annotations as anchors


# Template 

## Identifying gene expression programs of cell-type identity and cellular activity with single-cell RNA-Seq
[@doi:10.7554/eLife.43803]

### Highlights
Identifying gene expression programs underlying both cell-type identity and cellular activities (e.g. life-cycle processes, responses to environmental cues) is crucial for understanding the organization of cells and tissues. Although single-cell RNA-Seq (scRNA-Seq) can quantify transcripts in individual cells, each cell’s expression profile may be a mixture of both types of programs, making them difficult to disentangle.

Consensus non-negative matrix factorization (cNMF) accurately refines cell types and identifies both expected (e.g. cell cycle and hypoxia) and novel activity programs.

Individual cells may express multiple GEPs but we only detect cellular expression profiles that reflect their combination, rather than the GEPs themselves.

We group these into two broad classes of expression programs that can be detectable in scRNA-Seq data: 
(1) GEPs that correspond to the identity of a specific cell type such as hepatocyte or melanocyte (identity programs)
(2) GEPs that are expressed independently of cell type, in any cell that is carrying out a specific activity such as cell division or immune cell activation (activity programs).

We hypothesized that we could infer activity GEPs directly from variation in single-cell expression
profiles using matrix factorization.

We refer to the second matrix as a ‘usage’ matrix as it specifies how much each GEP is ‘used’ by each cell in the dataset (Stein-O’Brien et al., 2018) (Figure 1A). Unlike hard clustering, which reduces all cells in a cluster to a single shared GEP, matrix factorization allows cells to express multiple GEPs.

GEPs corre- sponding to different phases of the cell cycle are examples of widespread activity programs and are well-known to confound identity (cell type) program inference in scRNA-Seq data.

We distinguished between identity and activity programs by using the fact that activity programs can occur in multiple diverse cell types while identity programs represent a single-cell type.

Most cells had high usage of just a single GEP, which is consistent with expressing just an identity program.

cNMF was further able to refine cell types by disentangling the contributions of identity and activ-
ity programs to the gene expression of cells. For example, we found that a cell cluster labeled in Quadrato et al. (2017) as ‘proliferative precursors’, based on high expression of cell-cycle genes, is composed of multiple cell types including immature muscle and dopaminergic neurons.

Identification of activity programs in neurons of the visual cortex.

In this study, we distinguished between cell type (identity) and cell type independent (activity) gene expression programs (GEPs) to motivate our use of matrix factorization, which represents cells as lin- ear combinations of multiple GEPs.

However, we note that some biological programs are not neatly classified as either identity or activity GEPs. For example, cell states reflecting oncogenic transforma- tion, or a cell’s position along a morphological gradient blur the distinction between identity and activity. In addition, stochastic fluctuations in individual transcription factors could result in coordi- nated gene expression changes (Thattai and van Oudenaarden, 2001) and might be better described as a third program category, rather than as an identity or activity GEP. While the identity/ activity distinction might not be appropriate in every case, matrix factorization should, in principle, be appropriate for representing all gene expression states that can be reasonably approximated as a linear mixture of programs.

Beyond simply discovering activity programs, cNMF clarified the underlying cell types in these datasets by disentangling activity and identity programs from the mixed single-cell profiles.

A more fundamental limitation of matrix factorizations, including cNMF, is the built-in assumption
that cells can be modeled as linear combinations of GEPs.


### Comments

# Experimental single-cell research 

## Single-cell RNA-seq reveals new types ofhuman blood dendritic cells, monocytes, and progenitors
[@doi:10.1126/science.aah4573]
### Highlights

Atlas of human blood dendritic cells and monocytes

Dendritic cells (DCs) and monocytes consist of multiple specialized sub- types that play a central role in pathogen sensing, phagocytosis, and antigen presentation. However, their identities and interrelationships are not fully understood, as these populations have historically been definedbyacombination of morphology, physical properties, localiza- tion, functions, developmental origins, and expression ofa restricted set of surface markers.


Single-cell RNA sequencing of~2400cells isolated from healthy blood donors and enriched for HLA-DR+ lineage− cells

six DC subtypes
four monocyte subtypes, and thus revise the taxonomy of these cells

DC subset, representing 2 to 3% of the DC populations across all __10 donors__ tested, characterized by the expression ofAXL, SIGLEC1, and SIGLEC6 antigens, named AS DCs.

This split is further reflected through AS DC gene expression signatures spanning aspectrumbetween cDC-like andpDC- like gene sets. Although AS DCs share prop- erties with pDCs, they more potently activate T cells. This discovery led us to reclassify pDCs as the originally described “natural interferon- producing cells (IPCs)” with weaker T cell pro- liferation induction ability.

A new subdivision within the CD1C+ DC subset: one defined by a major histocompatibility complex class II–like gene set andonebyaCD14+ monocyte–like prominent gene set


The discovery of AS DCs withinthe traditionally definedpDCpopulation explains manyofthe cDC properties previously assigned to pDCs, highlighting the need to re- visit the definition of pDCs


it is unclear how many DC subtypes exist, how they are related to each other, and how they differ from other mononuclear phagocytes.

In the blood, DC subtypes include CD11C+ con- ventional DCs (cDCs), consisting of either CD141+ or CD1C+ cells, and plasmacytoid DCs (pDCs), consisting of CD123+ cells.


we developed an experimental and computational strategy to 
(i) perform single-cell RNA sequencing on DCs and monocytes derived from a single healthy individual;
(ii) identify clusters of cells that are similar to each other; 
(iii) find discriminative markers per cluster; 
(iv) prospectively isolate cells corresponding to key clusters using newly identified surface markers; 
(v) validate the identity of the sorted cells using scRNA-seq; (vi) confirm the existence of these cell types in up to 10 independent healthy individuals; and 
(vii) perform functional analyses for selected cell types.
Single-cell

Ficoll-enriched cells that were isolated by fluorescence-activated cell sorting (FACS) (Fig. 1A) excluding cells expressing markers of B, T, and NK cells (6).

For DCs, we sampled LIN–HLA-DR+CD14– cells across the CD11C+ fraction (to enrich for CD141+ and CD1C+ cDCs) and the CD11C– fraction (to enrich for CD123+ pDCs)

deep scRNA-seq using a modified Smart-Seq2 protocol

LIN–HLA- DR+CD14– population

We defined six cell clusters within the LIN–HLA- DR+CD14– population using unsupervised analy- sis that did not rely on known markers of DCs

Up to five top discriminators are listed next to each cluster; number in brackets refers to AUC value.

Axl (1.00)
SIGLEC6 + (0.97)

Several markers commonly used to identify pDCs (e.g., IL3RA/CD123, CLEC4C/CD303) were also expressed in the population defined by the DC5 cluster, leading us to define a new combination of markers that distinguish pDCs from the DC5 population. 

Discovering monocyte subsets and their relationships to DC subsets. The two largest clusters, Mono1 and Mono2, contained the CD14++CD16– (“classical”) and CD14+CD16++ (“nonclassical”) monocytes, respectively

Population emerged from the unbiased cluster analysis (cluster DC5; Fig. 1), definedbycoexpressionofuniquemarkers (e.g., AXL, SIGLEC1, SIGLEC6,and CD22/SIGLEC2) (Fig. 4A, fig. S5A, and tables S1 and S2). Flow cytometry analysis of peripheral blood mono- nuclear cells (PBMCs) from 10 independent do- nors confirmed the existence of AXL+SIGLEC6+ cells (“AS DCs”)

AS DCs exhibited a spectrum of states based on gene expression (Fig. 4D) defined by cells enriched for a pDC-like signature (e.g., IL3RA, IGJ, NRP1, MZB1) and cells enriched for a cDC- like signature (IFI30, ITGAX, LY86, GLIPR2, FGR, LYZ, ENTPD1).

Taken to- gether, our data suggest that AXL+SIGLEC6+ DCs are related but not identical to cDCs or pDCs.

When we seeded cultures with pDCs and CD1C+ and CLEC9A+ DCs, we found that they generally retained the same phenotype throughout the differentiation assay. After 7 days in culture, we observed cells with high levels of CD1C (frequency 40 to 50%, n =6 donors) and rare cells with surface CLEC9A and CADM1 (0.5 to 0.8%) expression (Fig. 6F)

Cells differentiated from AS DCs did not express BATF3 (a transcription factor required for terminal differentiation of CLEC9A+ DCs), CADM1,or XCR1, which are key CLEC9A+ DC discriminative markers (table
 
We found that AS DCs did not divide during the transition into CD1C+ DCs.

We leveraged our human DC atlas to compare pathogenic cells driving blastic plasmacytoid dendritic cell neoplasm (BPDCN), a rare and ag- gressive hematological malignancy previously known as NK cell leukemia/lymphoma.

DCs and monocytes are defined according to a combination of molecular markers, functional properties, and ontogeny (39). However, it remains unclearwhether the expression ofexistingmarkers tracks with themore complex internal states of cells.

Previous studies classified human blood DCs into one pDC and twocDC populations. Our study identifies six DC populations.

AS DCs most closely resemble CD1C+ cDCs in basic functional properties and expres- sion signatures, they are likely to have distinct functions because they localize to the T cell zone of tonsils and express several lectins, which rec- ognize diverse glycans, andAXL,which interacts with apoptotic cells and Zika virus


Furthermore, a murine study identified nonca- nonical CX3CR1+CD8a+ cDCs (nc-cDCs), which ex- press pDC and cDC signatures (e.g., CX3CR1, CD11c, and MHCII), do not produce IFN-a,and activate T cell proliferation (47, 48). Interestingly, pDCs and nc-cDCs require E2-2/TCF4 to develop, and reduced levels ofE2-2 lead to higher ID2 and expression of cDC genes (18, 47, 48). Consistent with this finding, we observed E2-2/TCF4 ex- pression in human pDCs (Fig. 5A), with decreas- ing levels of E2-2/TCF4 and increasing levels of ID2 as AS DCs transition to CD1C+ DCs (fig. S5, C to F). These findings suggest that AS DCs are similar to human CD2hi pDCs andmurine nc-cDCs.

More generally, our use of the DC atlas to understand BPDCN cells illustrates how single-cell analysis can pin- point relationships of diseased cells to healthy cells.

Nonetheless, some DC/monocyte subtypes were likely missed because they do not express MHC class II at rest, can onlybe defined bynon-RNAmolecules, are distinguished by low-abundance transcripts, or are onlypresent during inflammation, disease, or within tissues. To build a comprehensive im- mune cell atlas, future studies will need to address these challenges as well as localize these cell types within lymphoid and nonlymphoid tissues.


Healthy donors were recruited from the Boston-based PhenoGenetic project, a resource of healthysubjectsthatare recontactablebygeno- type (57), and the Newcastle community. Individ- uals were excluded if they had a history ofcancer, allergies, inflammatory disease, autoimmune dis- ease, chronic metabolic disorders, or infectious disorders. All healthy donors were nonsmokers, had a normal BMI and normal blood pressure, and were between 25 and 40 years of age.




### Comments



# Template 

## Title
[citation]
### Highlights



### Comments